pkg/okteto/namespace.go

```diff
@@ -121,12 +121,10 @@ func (c *namespaceClient) Delete(ctx context.Context, namespace string) error {
 	var mutation struct {
 		Space struct {
 			Id graphql.String
-		} `graphql:"deleteSpace(id: $id, force: $force)"`
```

Comment:
@teresaromero I've removed this as I realize that it could fail if the Okteto instance is not updated. As the default value is false, we can omit it until we include the `--force` parameter in these commands

Reply:
can we leave a TODO comment ?

Reply:
I created a ticket https://github.com/okteto/app/issues/5340 to keep track of it but I wouldn't add a TODO comment. It wasn't a requirement in the feature document and it is an optional parameter, so it is something we might be leave like that forever if we don't see value in adding that option in the CLI.

Reply:
https://github.com/okteto/okteto/pull/3312